src/app/modules/browse/browse-controller.js

```diff
@@ -140,9 +140,29 @@
             }
             return _localize(message, params);
         }
+        function setHiddenBooksMessages() {
+            if ($scope.allLicenses) {
```

Comment:
I don't think we need to change that at this point. I was assuming we had it and it would be simpler to treat both the same. It would be fine for that message not to support {count}.